examples/monorepo-frontend/.structurelint.yml

```
@@ -0,0 +1,97 @@
+  # Test location (co-located)
+  test-location:
+    allow-adjacent: true
+
+  # Ensure TypeScript configs
+  file-existence:
+    "apps/admin/tsconfig.json": "TypeScript config required"
+    "apps/customer/tsconfig.json": "TypeScript config required"
+    "packages/design-system/tsconfig.json": "TypeScript config required"
+
+  # Package.json for each app/package
+  file-existence:
```

YAMLlint (1.37.1)

[error] duplication of key "file-existence" in mapping

(key-duplicates)